README.rst

```diff
@@ -1,7 +1,6 @@
 .. -*- mode: rst -*-
 
-
-|Travis|_ |Azure|_ |Circle|_ |Codecov|_ |Zenodo|_
+|Travis|_ |Azure|_ |Circle|_ |Codecov|_ |PyPi|_ |conda-forge|_ |Zenodo|_
```

Comment:
Rich preview for those curious

https://github.com/mne-tools/mne-python/blob/858970e1bb65fd539ef795d10e22f018cc7f314d/README.rst